django_wikidata_api/tests/test_models.py

```diff
@@ -16,20 +16,23 @@ class WikidataItemBaseTests(TestCase):
     def setUp(self):
         """ Setup WikidataItemBase tests. """
         self.test_item = WikidataItemBase()
-        self.test_item_full = WikidataItemBase(label="Test Item", main="Q123", id=123, alt_labels=["Test"])
+        self.test_item_full = WikidataItemBase(label="Test Item", description="Some Test Description", main="Q123",
+                                               id=123, alt_labels=["Test"])
         self.mocked_query_response = {
             'results': {
                 'bindings': [
                     {
                         'main': {'type': 'uri', 'value': 'http://www.wikidata.org/entity/Q123'},
                         'mainLabel': {'xml:lang': 'en', 'type': 'literal', 'value': 'Test Item'},
                         'label': {'xml:lang': 'en', 'type': 'literal', 'value': 'Test Item'},
+                        'description': {'xml:lang': 'en', 'type': 'literal', 'value': 'Some Test Description'},
```

Comment:
In this file, I'm making sure the full integration with Wikidata will successfully parse the descriptions in the Wikidata Query Service response format